### PR TITLE
=lib Add return type for Exception#handling

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: src/library/scala/util/control/Exception.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import scala.language.implicitConversions
  *  Each class is independently composable.
  *
  *  This class differs from [[scala.util.Try]] in that it focuses on composing exception handlers rather than
- *  composing behavior.  All behavior should be composed first and fed to a [[Catch]] object using one of the
+ *  composing behavior.  All behavior should be composed first and fed to a [[scala.util.control.Exception.Catch]] object using one of the
```
Comment (NthPortal) [inline src/library/scala/util/control/Exception.scala]: Might still want the shorter display text

```suggestion
 *  composing behavior.  All behavior should be composed first and fed to a [[scala.util.control.Exception.Catch Catch]] object using one of the
```

Comment (He-Pin) [reply src/library/scala/util/control/Exception.scala]: Okay, but the shorter one seems an error.

**Comment by SethTisue** [comment]
congrats on your first scala/scala PR :-)

**Comment by He-Pin** [comment]
Thanks for the guidance

**Pull request merged**